state/protocol/inmem/dkg_test.go

```diff
@@ -0,0 +1,59 @@
+package inmem_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol"
+	"github.com/onflow/flow-go/state/protocol/inmem"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// TestDKGv0 tests that the [inmem.DKG] is backward compatible with the v0 DKG protocol model.
+// We test this by creating a [inmem.DKG] instance from a v0 [flow.EpochSetup] and [flow.EpochCommit]
+// and verifying that the DKG methods return the expected values.
+// TODO(EFM, #6794): Remove this once we complete the network upgrade
```

Comment:
are we going to  _entirely_ remove this test? I think it wouldn't hurt to still keep the tests for the [DKG](https://github.com/onflow/flow-go/blob/0b12b6342a5e526a6860edb07d789748e3c03f4a/state/protocol/inmem/dkg.go#L14) struct

Reply:
Makes sense, I will leave it.